<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LIAEngineDialog</class>
 <widget class="QDialog" name="LIAEngineDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>126</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="modeLabel">
     <property name="text">
      <string>LIA operation mode</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="modeBox">
     <item>
      <property name="text">
       <string>4 ch LIA @ 5kHz</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3 ch LIA @ 10kHz</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2ch LIA @ 25kHz</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1ch LIA @ 50kHz</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Frequency sweep</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="freqHighLabel">
     <property name="text">
      <string>Frequency sweep high frequency</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="freqHighBox">
     <item>
      <property name="text">
       <string>10Hz</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20Hz</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>50Hz</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>100Hz</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10kHz</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>200Hz</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1kHz</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2kHz</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5kHz</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>500Hz</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20kHz</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>50kHz</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="freqLowLabel">
     <property name="text">
      <string>Frequency sweep low frequency</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="freqLowBox">
     <item>
      <property name="text">
       <string>10Hz</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20Hz</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>50Hz</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>100Hz</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>200Hz</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>500Hz</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1kHz</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2kHz</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5kHz</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10kHz</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20kHz</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>50kHz</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="confirmBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>confirmBox</sender>
   <signal>accepted()</signal>
   <receiver>LIAEngineDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>confirmBox</sender>
   <signal>rejected()</signal>
   <receiver>LIAEngineDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
